src/lib/shared/api/supabase.ts

```diff
@@ -0,0 +1,8 @@
+import { createClient } from '@supabase/supabase-js';
+import constants from '../constants';
```

Comment:
suggestion (non-blocking): could we name it more specifically than `constants`? something like `supabase-credentials`